src/locales/de/pdf.json

```
@@ -0,0 +1,28 @@
+{
+	"calendar": {
+		"header": {
+			"week-number": "W#",
```

In German there actually exists the abbreviation of KW, meaning Kalenderwoche
```suggestion
			"week-number": "KW",
```